groups/templates/groups/discussion_thread_base.html

```diff
@@ -20,8 +20,15 @@
                 <ul id="{{ comment.get_pagejump_anchor }}">
                     <li>
                         {{ comment.user }} wrote at <a href="{{ comment.get_pagejump }}">{{ comment.date_created }}</a>:
+                        {% if comment.user_may_delete %}(<a href="{% url 'comment-delete' pk=comment.pk %}">delete this comment</a>){% endif %}
+                    </li>
+                    <li>
+                        {% if comment.is_deleted %}
```

Comment:
Sure, that works, although it does look weird that it's then used to display `(Post deleted)`?